package.json

```diff
@@ -213,6 +212,7 @@
     "html-webpack-plugin": "^5.3.1",
     "husky": "6.0.0",
     "imports-loader": "^2.0.0",
+    "isomorphic-fetch": "^3.0.0",
```

Comment:
Necessary in tests, since `node-fetch` doesn't have the `Request` and `Response` browser objects.